ui/lib/pki/addon/components/pki-key-details.hbs

```diff
@@ -48,12 +48,9 @@
 
 <div class="box is-fullwidth is-sideless is-paddingless is-marginless">
   {{#each @key.formFields as |attr|}}
-    {{#let (get @key attr.name) as |value|}}
-      <InfoTableRow
-        @label={{capitalize (or attr.options.detailsLabel attr.options.label (humanize (dasherize attr.name)))}}
-        @value={{value}}
-        @alwaysRender={{true}}
-      />
-    {{/let}}
+    <InfoTableRow
+      @label={{capitalize (or attr.options.detailsLabel attr.options.label (humanize (dasherize attr.name)))}}
+      @value={{get @key attr.name}}
+    />
```

Comment:
Was the `alwaysRender` arg not needed?

Reply:
`keyBits` is an attribute available when generating a key, but is not currently included in the `GET` response for a key. (There's discussion to add it, but it's not looking likely) Since it will always be empty, I opted to remove `alwaysRender=true` so it can still appear in the create form, but won't render the row in the details page